web/src/i18n/locales/en/activities/overview.yaml

```
@@ -44,5 +45,5 @@ alternatives:
     If substantive changes have been made, include statement describing alternative
     considerations made before selecting implementation strategy.
   reminder: >-
-    If not, including statement of alternative considerations from previous versions
-    of IAPD or SMHP.
+    If substantive changes have not been made, include a statement of alternative considerations from previous version
```

```suggestion
    If substantive changes have not been made, include a statement of alternative considerations from the previous version
```
the? a? Feels like we need an article there.

---

added "the" good catch.